docs/vmagent.md

```diff
@@ -1893,6 +1893,10 @@ See the docs at https://docs.victoriametrics.com/vmagent/ .
   -maxInsertRequestSize size
      The maximum size in bytes of a single Prometheus remote_write API request
      Supports the following optional suffixes for size values: KB, MB, GB, TB, KiB, MiB, GiB, TiB (default 33554432)
+  -maxLabelValueLen int
+     The maximum length of label values in the accepted time series. Series, with longer label values are dropped. In this case the `vm_series_dropped_total{reason="too_long_label_value"}` metric at /metrics page is incremented (default 4096).
```

Comment:
Please also mention the limits is applied on series after `-remoteWrite.relabelConfig`.